src/ConfigCountryModal.sass

```diff
@@ -0,0 +1,8 @@
+@import './App.sass'
+
+
+.ant-btn
```

Comment:
This runs on every button, open the modal and check out the buttons there.
You need to add a specific className/ID